src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -3152,8 +3149,9 @@ Real64 EvaluateIntHcModels(EnergyPlusData &state, int const SurfNum, HcInt const
     auto const &surfIntConv = state.dataSurface->surfIntConv(SurfNum);
 
     int const ZoneNum = thisSurface.Zone;
+    int const spaceNum = thisSurface.spaceNum;
     Real64 const Tsurface = state.dataHeatBalSurf->SurfInsideTempHist(1)(SurfNum);
-    Real64 const Tzone = state.dataZoneTempPredictorCorrector->zoneHeatBalance(ZoneNum).MAT;
+    Real64 const Tzone = state.dataZoneTempPredictorCorrector->spaceHeatBalance(spaceNum).MAT;
```

Comment:
OK, got the zone temp from the space structure now...is that just a renamed struct or a different struct entirely?

Reply:
Different struct entirely.